hw3.1/style.css

```diff
@@ -0,0 +1,125 @@
+.banner{
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+}
+header{
+    background-color: rgb(154, 9, 9);
+    height: 50px;
+}
+.images{
+    background-color: black;
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+}
+.big{
+    width: 220px;
+    height: 160px;
+}
+div.images{

```

Comment:
You can use here just class selector without tag (div.images -> .images)

Reply:
+